clinica/pipelines/statistics_volume/template_model_contrast.m

```diff
@@ -0,0 +1,17 @@
+spm('defaults','pet');
```

Comment:
For all Matlab files, could you add a comment on how you obtained these files on the header?

e.g. `% File automatically generated using batch mode after chosing SPM > MRI/PET > Estimate`

Reply:
Done